lib/toPaths.js

```diff
@@ -1,13 +1,18 @@
 var isArray = Array.isArray;
 var typeOfObject = "object";
+var typeOfString = "string";
+var typeOfNumber = "number";
+var MAX_SAFE_INTEGER = 9007199254740991; // Number.MAX_SAFE_INTEGER in es6
+var abs = Math.abs;
+var safeNumberRegEx = /^(0|(\-?[1-9][0-9]*))$/;
```

Comment:
Falcor treats numbers and string representations of numbers interchangeably, so we need to make sure that any strings we convert to integers roundtrip convert back to the same string, i.e. the user input `'03'` should not be converted to the string `'3'` by the time it hits the get handler.